packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCPP/SampleModuleCPP.h

```diff
@@ -113,6 +119,51 @@ struct SampleModuleCPP {
   }
 
 #pragma endregion
+
+#pragma region Synchronous Methods
+
+  REACT_SYNC_METHOD(SyncReturnMethod);
+  double SyncReturnMethod() noexcept {
+    DebugWriteLine("SyncReturnMethod");
+    return M_PI;
+  }
+
+  REACT_SYNC_METHOD(SyncReturnMethodWithArgs);
+  double SyncReturnMethodWithArgs(double arg) noexcept {
+    DebugWriteLine("SyncReturnMethodWithArgs", arg);
+    return M_PI;
+  }
+
+#pragma endregion
+
+#pragma region Events
+
+  REACT_EVENT(TimedEvent, "TimedEventCPP");
+  std::function<void(int)> TimedEvent;
+
+#pragma endregion
+
+ public:
+  SampleModuleCPP() {
+    _timer = winrt::Windows::System::Threading::ThreadPoolTimer::CreatePeriodicTimer(
+        [this](const winrt::Windows::System::Threading::ThreadPoolTimer) noexcept {
+          if (TimedEvent) {
+            TimedEvent(++m_timerCount);
+          }
+        },
+        std::chrono::milliseconds(TimedEventIntervalMS));
+  }
+
+  ~SampleModuleCPP() {
+    if (_timer) {
+      _timer.Cancel();
+    }
+  }
+
+ private:
+  winrt::Windows::System::Threading::ThreadPoolTimer _timer = nullptr;
```

Comment:
>_ [](start = 53, length = 1)

please add m_ prefix: in C++ all names that start with '_' or '__' are reserved for the standard library or the language. User code must not have such names.

Reply:
Fixed.